### PR TITLE
APP-4531 Add missing descriptions in Docs tab of StoryBook

### PR DESCRIPTION
**Pull request by symphony-jean-michael**
**Jira ticket**
https://perzoinc.atlassian.net/browse/APP-4531

**Changes**
Add missing dependency to 'react-docgen-typescript-loader' package because the descriptions were not correctly loaded from our components.
- Update TimePicker description:
- Code cleanup in other stories

## TimePicker

**Demo without the fix**
Screenshot 2021-09-29 at 16 06 57

**Demo with the fix**
Screenshot 2021-09-29 at 16 05 30

## Other components, for example: Button

**Demo without the fix**
Screenshot 2021-09-29 at 16 11 15

**Demo with the fix**
Screenshot 2021-09-29 at 16 11 29

### CODE REVIEW

**Review by AliciaSymphony — APPROVED** [review]
Nice!

File: src/components/time-picker/TimePicker.tsx [hunk 1]
```diff
@@ -434,4 +415,13 @@ const handleFocus = (event) => {
   }
 };
 
+TimePicker.defaultProps = {
+  disabledTimes: [],
+  format: getUserFormat(),
+  min: '00:00:00',
+  max: '23:59:59',
+  step: STEP.DEFAULT_STEP_VALUE,
```
Comment (symphony-adnane) [inline src/components/time-picker/TimePicker.tsx:423]: is step an enum?

**Review by symphony-adnane — APPROVED** [review]

File: src/components/time-picker/TimePicker.tsx (hunk 1 above)
Comment (symphony-jean-michael) [reply src/components/time-picker/TimePicker.tsx:423]: Yes it is

Comment (symphony-jean-michael) [reply src/components/time-picker/TimePicker.tsx:423]: The default value is taken from an enum, but it is a number in seconds. (min: 600, default: 900, max 43200). The step value can be anything between min and max

**Pull request merged**